src/services/connect.ts

```diff
@@ -0,0 +1,36 @@
+import { StellarSDK } from '../api/stellarSdk';
+import type { Keypair } from 'stellar-sdk';
+
+const encryptKey = (key: string, salt: number) => {
+    const encodedData = btoa(key);
```

Comment:
don't use `btoa`, give https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/encrypt a look, seems to work everywhere except IE11 for which we can polyfill it later on